pkg/templates/html/attendees-limit-reached.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	  <head>
+		  <meta http-equiv="Refresh" content="1; url={{.}}" />
+      <title>Big Blue Button - Attendees limit reached</title>
+	  </head>
+	  <body>
+      <h1>The limit of attendees for this frontend was reached.</h1>
```

Comment:
We probably shouldn't talk about b3scale terminology in user-facing dialogues. "Frontend" is not something a user is likely to understand. Maybe drop the word "frontend", find a user friendly wording, or provide an error string that includes "frontend" and suggest to the user to report this.

Reply:
I've removed the mention of "frontend" for now. In a production setup it would probably good to replace this template with a template with more information as this is something a normal user could see.

Reply:
I guess having a way to override templates (i.e. by having an external template path or something) would be a nice additional feature.

Reply:
Yes that would be good. Something we could look into for a future PR.